templates/supplier_app/taxpayer-details.html

```diff
@@ -372,7 +372,7 @@ <h6 class="card-subtitle text-muted">{% trans "Country"%}</h6>
                     {% if is_AP %}
                     <div class="mr-auto">
                         {% if taxpayer.taxpayer_state == 'APPROVED' %}
-                        <a class="btn btn-danger " href="{% url 'pdf-web' taxpayer_id=taxpayer.id %}">
+                        <a class="btn btn-danger " target="_blank" href="{% url 'pdf-web' taxpayer_id=taxpayer.id %}">
```

Comment:
nit: Just a comment. Do you know this?
https://stackoverflow.com/questions/14945242/django-query-single-underscore-behaving-like-double-underscore
You could do `taxpayer_id` instead of `taxpayer.id`